docker/ubuntu20.04-sydr-fuzz/Dockerfile

```diff
@@ -29,6 +29,8 @@ RUN curl https://sh.rustup.rs -o rustup.sh && chmod +x rustup.sh && \
 
 ENV PATH="/root/.cargo/bin:${PATH}"
 
+RUN cargo install casr
```

Comment:
We'd better supply pre-build Casr in sydr-fuzz zip. It would be really hard to rebuild this docker on every Casr release.